chapters/00-why-should-i-learn.md

```diff
@@ -54,6 +54,12 @@ nvim plugins are the first to "evolve" when a new tech is coming to the IDE worl
 
 You can take a look at the [awesome nvim plugin list](https://github.com/rockerBOO/awesome-neovim). I recommend to "watch" to get updates about new plugins.
 
+#### Be comfortable
+nvim is basically a framework for an editor, without plugins it can't do much but it gives the user the ability to configure it's editor in a way he is most comfortable with.
```

Comment:
```suggestion
nvim is basically a framework for an editor, it can't do much without plugins but it gives the user the ability to configure the editor in a way they're most comfortable with.
```